fhir-persistence/src/main/java/com/ibm/fhir/persistence/FHIRPersistence.java

```diff
@@ -130,4 +132,13 @@
     default boolean isDeleteSupported() {
         return false;
     }
+
+    /**
+     * Generates a logical identity.
+     * 
+     * @return logical identity
+     */
+    default String getLogicalId() {
```

Comment:
I wonder if the method name should be `generateResourceId()` or some such to make it more clear that its no a logical id for the FHIRPersistence object but instead is used to generate resource ids.

Reply:
I renamed to generateResourceId()